apps/frontend/app/(client)/(code-editor)/_components/TestcasePanel/useTestResults.ts

```diff
@@ -60,6 +61,10 @@ export const useTestResults = () => {
     setIsTesting,
     stopPolling
   } = useTestPollingStore((state) => state)
+  const [getTestcases] = useTestcaseStore(
+    useShallow((state) => [state.getTestcases])
+  )
+  const testcases = getTestcases()
```

Comment:
```suggestion
  const testcases = useTestcaseStore(
    useShallow((state) => state.getTestcases())
  )
```

혹시 이렇게 작성해도 무한루프 발생하나요??

Reply:
넵.. 아쉽게도

Reply:
ㅠㅠ